naas_python/utils/domains_base/authorization.py

```diff
@@ -149,10 +173,10 @@ def jwt_token(self):
 class NaasSpaceAuthenticatorAdapter(IAuthenticatorAdapter):
     def __init__(
         self,
-        port=8080,
+        port=38745,
         trade_url="https://auth.naas.ai/bearer/workspace/longlived",
-        timeout=5,  # 60,
-        login_url="https://naas.ai?cli_token=generate_token",
+        timeout=60,  # 60,
+        login_url=os.environ.get('LOGIN_URL', "https://naas.ai?cli_token=generate_token"),
```

Comment:
If we leave this as something we can update by env vars, I suggest that we keep a standard of anything that can be overwritable would have the config name happened with `NAAS_.` So that this is not mistaken updated by something the user might have in its environments already

Reply:
That is a good feedback. I will update that